src/command/metar/RunwayCommand.ts

```diff
@@ -13,7 +13,7 @@ import { ICommand } from "../metar";
 export class RunwayCommand implements ICommand {
   #genericRegex = /^(R\d{2}\w?\/)/;
   #runwayMaxRangeRegex = /^R(\d{2}\w?)\/(\d{4})V(\d{3,4})([UDN])?(FT)?/;
-  #runwayRegex = /^R(\d{2}\w?)\/([MP])?(\d{4})([UDN])?(FT)?$/;
+  #runwayRegex = /^R(\d{2}\w?)\/([MP])?(\d{4})([UDN])?(FT)?(\/([UDN]))?$/;
```

Comment:
If proceeded by "FT", is "/" always required before trend? Its possible this code has always been flawed for North American METARs.

Maybe a cleaner/stricter way to write would be like this:

```
^R(\d{2}\w?)\/([MP])?(\d{4})?(?:(FT)\/([UDN])|([UDN]))?$
```

This would prevent the following invalid runway command from matching:

```
R36/4000UFT/U
```

Reply:
Yeah I was not sure what other formats are out there so I did the "easy" route and just append the canadian format.

Reply:
In general I try to lean to being strict to avoid undefined/untested behavior. If my regex is easy to integrate then lets go with that!

Reply:
I am not sure if its always required. I wouldn't be surprised if this is one of the things where a clear standard is not followed. I can change it if you want but I am no expert in regards of if this will cause unintended issues because some country uses RVR without the `/`

Reply:
No worries, for this project I am 100% OK with breaking things if there is no test coverage for it.

Actually I encourage it because everything should be tested to avoid regressions in the first place!

Reply:
Ok your regex didn't work but I've re-created a more strict regex that passes all tests. Also added a missing dedicated parser test for this specific format.